.eslintrc.js

```diff
@@ -12,35 +12,38 @@ module.exports = {
-        'eol-last'                          : ['error', 'always'],
-        'func-names'                        : ['error', 'never'],
-        'keyword-spacing'                   : ['error', { after: true }],
-        'lines-between-class-members'       : ['error', 'always', { exceptAfterSingleLine: true }],
-        'no-extra-semi'                     : 'error',
-        'no-console'                        : 'error',
-        'no-else-return'                    : ['error', { allowElseIf: true }],
-        'no-multi-assign'                   : 0,
-        'no-param-reassign'                 : ['error', { props: false }],
-        'no-restricted-globals'             : 0,
-        'no-script-url'                     : 0,
-        'no-trailing-spaces'                : ['error', { skipBlankLines: true }],
-        'object-curly-spacing'              : ['error', 'always', { arraysInObjects: true, objectsInObjects: true }],
-        'one-var'                           : ['error', { initialized: 'never', uninitialized: 'always' }],
-        'prefer-destructuring'              : 0,
-        'space-in-parens'                   : ['error', 'never'],
-        'space-infix-ops'                   : 'error',
-        'space-unary-ops'                   : 'error',
-        'no-multiple-empty-lines'           : ['error', { 'max': 1, 'maxEOF': 1 }],
+        camelcase                       : 0,
+        semi                            : ['error', 'always'],
+        'array-callback-return'         : 0,
+        'brace-style'                   : ['error', '1tbs', { allowSingleLine: true }],
+        'eol-last'                      : ['error', 'always'],
+        'func-names'                    : ['error', 'never'],
+        'keyword-spacing'               : ['error', { after: true }],
+        'lines-between-class-members'   : ['error', 'always', { exceptAfterSingleLine: true }],
+        'no-extra-semi'                 : 'error',
+        'no-console'                    : 'error',
+        'no-else-return'                : ['error', { allowElseIf: true }],
+        'no-multi-assign'               : 0,
+        'no-param-reassign'             : ['error', { props: false }],
+        'no-restricted-globals'         : 0,
+        'no-script-url'                 : 0,
+        'no-trailing-spaces'            : ['error', { skipBlankLines: true }],
+        'object-curly-spacing'          : ['error', 'always', { arraysInObjects: true, objectsInObjects: true }],
+        'one-var'                       : ['error', { initialized: 'never', uninitialized: 'always' }],
+        'prefer-destructuring'          : 0,
+        'space-in-parens'               : ['error', 'never'],
+        'space-infix-ops'               : 'error',
+        'space-unary-ops'               : 'error',
+        'no-multiple-empty-lines'       : ['error', { 'max': 1, 'maxEOF': 1 }],
+        'prefer-exponentiation-operator': 0,
+        'prefer-object-spread'          : 0,
```

Comment:
Turned off `prefer-exponentiation-operator` & `prefer-object-spread` since this code base is legacy and it only supports ES2015 (E6) syntax.
Please let me know if any concerns.